### PR TITLE
[Map] Medical revamp and major changes.

### LINKED ISSUE #4225: Diner Wall Chairs
image
image

There are no tables.
Only walls.
We eat with walls now.
No tables.
With walls.

### LINKED ISSUE #4311: Wiring to the Surface Level's roof does not line up, surface level solar maintenance not powered.
The Surface Solar Maintenance, the one above the cargo bay, never connects to the powernet. I followed the cables and removed the lattice on the south end of the roof where the cable turns down. Looking down I saw a steel wall below me. I went to the surface level and verified by moving up and down with a jetpack that the wire turns down into an exterior steel wall.

EDIT: Additionally, where the red main line runs past the yellow solar line just outside of the maintenance room, the two wires are actually registering as connected. So if the solars are online they will provide power straight into the main line in addition to just going to their SMES.

### PR DESCRIPTION
**Pull request by Juani2400**
- Complete remap of the Medical main level. Expect bugs and missing
equipment.
- Added a new isolation wing to Medical.
- Changed the nuke's position, to a much safer, new location.
- Added an alternative exit to the Bunker.
- Added a new store that only the HoP can grant access into.
-Detective and CSI's offices have been moved to a level above Security.
- New Security training area, for cadets and other uses you may find to
it.
- New transfer/escape shuttle. Not the final version, probably. Test of
acceptance from the community in progress. Not entirely convince by the
design.
- New Research-coloured folders (Sprite recolouring by Fire and Glory).
- Slightly remapped Kitchen. Tighter on space, big on table surface
area.
- New Research Office by Xenobiology, after a lot of requests.
- Vault entrance has been redesigned to fit a bit more with its purpose.
- A lot of bugfixes, missing stuff, and minor suggestions.

- Fixes #4311.
- Fixes #4225.

Album of images of some of the new areas:
https://imgur.com/a/HKjwt

EDIT: Merging this may break (hopefully not) some other map changes done before. I'll take care of adapting to those as soon as it is merged, as I have no way to compare the files before PR'ing them.

### CODE REVIEW

**Comment by Juani2400** [comment]
I don't know why it auto-requested review from Arrow. Ignore it.

**Comment by Lohikar** [comment]
Somehow travis took over 10 minutes to load the map and gave up. Does this work locally?

**Comment by Juani2400** [comment]
I'm pretty sure I know what it is. I'll fix it in approximately 14 hours
from now.

**Comment by skull132** [comment]
pictures please and thanks.

**Comment by skull132** [comment]
skull is fucking blind carry on.

File: maps/_common/areas/station/medical.dm [hunk 1]
```diff
@@ -150,3 +145,25 @@
 /area/medical/icu
 	name = "\improper Medical -  Intensive Care Unit"
 	icon_state = "cryo"
+
+/area/medical/triage
+	name = "\improper Medical -  Triage Room"
```
Comment (Lohikar) [inline maps/_common/areas/station/medical.dm]: Extra space.

File: maps/_common/areas/station/medical.dm [hunk 2]
```diff
@@ -150,3 +145,25 @@
...
+	icon_state = "medbay4"
+
+/area/medical/em_preop
+	name = "\improper Medical -  Emergency Pre-Op"
```
Comment (Lohikar) [inline maps/_common/areas/station/medical.dm]: Extra space.

File: maps/_common/areas/station/security.dm [hunk 3]
```diff
@@ -57,10 +57,17 @@
...
 	icon_state = "detective"
 
+/area/security/investigations
+	name = "\improper Security - Investigations Division"
+	icon_state = "detective"
+
 /area/security/range
 	name = "\improper Security - Firing Range"
 	icon_state = "firingrange"
-	flags = FIRING_RANGE
```
Comment (Lohikar) [inline maps/_common/areas/station/security.dm]: pls

File: maps/aurora/aurora-2_under-station.dmm [hunk 4]
```diff
@@ -1,5 +1,4 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
-//Model dictionary trimmed on: 18-11-2017 20:57 (UTC)
```
Comment (Lohikar) [inline maps/aurora/aurora-2_under-station.dmm]: ...why even commit this file if you made no changes to it.

File: code/modules/modular_computers/file_system/programs/app_presets.dm [hunk 5]
```diff
@@ -229,4 +228,17 @@
...
\ No newline at end of file
+	return _prg_list
+
+
+/datum/modular_computer_app_presets/ai
+    name = "ai"
+    display_name = "AI"
+    description = "A preset for the AI consoles."
+    available = 0
+/datum/modular_computer_app_presets/ai/return_install_programs()
```
Comment (Lohikar) [inline code/modules/modular_computers/file_system/programs/app_presets.dm]: There should be a newline before this line.

**Comment by Lohikar** [comment]
You seem to have based this on an old version of the map (and reverted some things in the process). Also has invalid paths.

**Comment by Lohikar** [comment]
Updated maps and resolved accidental reverts, changelog + icon changes need to be re-added by hand.

**Comment by Juani2400** [comment]
It should be fiiiiine now. Finally.
Thanks, Lohi.

File: html/changelogs/Juani-map.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - maptweak: "Complete remap of the Medical main level. Expect bugs and missing equipment."
```
Comment (BurgerLUA) [inline html/changelogs/Juani-map.yml]: Changelog seems a little long. I'm not a developer but they might complain about the size of it.

Comment (LordFowl) [reply html/changelogs/Juani-map.yml]: I am a developer and I am complaining about it. If push comes to shove the changelog can link somewhere that actually has pictures and such where a better image of what has changed can be perceived.

**Pull request merged**